scripts/content.js

```diff
@@ -96,27 +111,42 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
 
                   var EnvironmentScore=document.createElement('span');
                   EnvironmentScore.classList.add('score');
-                  EnvironmentScore.innerText=resp.result[index].EnvironmentScore;
-                  EnvironmentScore.style["background-color"] =resp.colourMap[resp.result[index].EnvironmentScore];
+
+                  if(resp.result[index].EnvironmentScore){
+                    EnvironmentScore.innerText=resp.result[index].EnvironmentScore;
+                    EnvironmentScore.style["background-color"] =resp.colourMap[resp.result[index].EnvironmentScore];
+
+                  }
+                  else{
+                    EnvironmentScore.innerText="TBD";
+                    EnvironmentScore.style["background-color"] =resp.colourMap[resp.result[index].EnvironmentScore];
+
+                  }
 
 
                   var EnvironmentScoret=document.createElement('span');
                   EnvironmentScoret.classList.add('scoret');
-                  EnvironmentScoret.innerText=" EnvironmentScore " ;
+                  EnvironmentScoret.innerText=" Environment Rate " ;
 
                   //OverallScore
                   var OS=document.createElement('div');
                   OS.classList.add('os');
 
                   var OverallScore=document.createElement('span');
                   OverallScore.classList.add('scoreo');
+
+                if(resp.result[index].OverallScore){
                   OverallScore.innerText=resp.result[index].OverallScore;
                   OverallScore.style["background-color"] =resp.colourMap[resp.result[index].OverallScore];
-
+                }
+                else {
+                  OverallScore.innerText="TBD";
+                  OverallScore.style["background-color"] =resp.colourMap[resp.result[index].OverallScore];
+                }
 
                   var OverallScoret=document.createElement('span');
                   OverallScoret.classList.add('scoret');
-                  OverallScoret.innerText= " OverallScore ";
+                  OverallScoret.innerText= " Overall Rate ";
 
                   //Link Page
                   var brandpagelink ="https://wellspent-ethical.herokuapp.com/#/Brand/"+brand[0];
```

Comment:
This no longer works in brands as the URL on the site must match the same capitalisation of the brand in the Airtable. 

So we need to change our approach.

Change it here: https://github.com/FACK1/wellSpent-chrome-extension/pull/5/files#diff-681a87337854a3455ada62b488bcb857R33
 So that we return brand name with the correct capitalisation: `return result.length > 0 ? [data.BrandName] : [];` 

And then when we compare here https://github.com/FACK1/wellSpent-chrome-extension/pull/5/files#diff-681a87337854a3455ada62b488bcb857R43, we just lowercase it then: ` if(productNodeText.includes(brand[0].toLowerCase())){`

We then keep this line as it is and it will link to the correct page in the site.

Reply:
Done :+1: